cli/cli/gateway.py

```diff
@@ -145,14 +168,22 @@ def get_routes(self) -> list[Route]:
             service_protocol = service["protocol"]
             service_url = f"{service_protocol}://{service_host}:{service_port}"
 
-            routes.append(
-                Route(
-                    relative_url=route_path,
-                    target=service_url,
-                    http_methods=http_methods,
-                )
+            r = Route(
+                relative_url=route_path,
+                target=service_url,
+                http_methods=http_methods,
             )
 
+            # Check if route has plugins installed
+            if route_id in route_plugins:
+                for p in route_plugins[route_id]:
+                    if p.get("name") == "jwt":
+                        r.jwt = True
```

Comment:
This conditional can be removed also, but it starts being horrible :( 
```
r.jwt |= p.get("name") == "jwt"
```

Reply:
Yes I would always prefer nesting over a more confusing one-liner. I'll add a comment that multiple plugin entries may exist for each route.